engelsystem/build.gradle

```diff
@@ -24,5 +25,8 @@ targetCompatibility = Config.compatibleJavaVersion
 compileKotlin {
     kotlinOptions {
         jvmTarget = Config.compatibleJavaVersion
+        freeCompilerArgs += [
+                "-Xopt-in=kotlin.RequiresOptIn"
```

Comment:
Do you really want to enable this for the whole project or only for the tests? If you only want this for the tests, maybe you want to wrap it in a 

```
test{...}
```
or 
```
subprojects {
    tasks.withType(Test) {
       ...
    }
}
```
block?

Reply:
I tried [`compileTestKotlin`](https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/jvm/build.gradle#L123) ...

``` gradle
compileTestKotlin {
    kotlinOptions {
        freeCompilerArgs += [
                "-Xopt-in=kotlin.RequiresOptIn"
        ]
    }
}
```

... but the Lint warning does not disappear for me in Android Studio Bumblebee | 2021.1.1 Patch 2.

Reply:
Nevermind, then leave it as it is :)